application/src/main/java/org/thingsboard/server/service/notification/channels/MicrosoftTeamsNotificationChannel.java

```diff
@@ -108,7 +109,7 @@ public void sendNotification(MicrosoftTeamsNotificationTargetConfig targetConfig
                 message.setPotentialAction(List.of(actionCard));
             }
         }
-
+        RestTemplateUtil.excludeJackson2XmlHttpMessageConverter(restTemplate);
```

Comment:
@irynamatveieva, please update this so we don't need to do this on each sendNotification request.